src/quickpicks/common.ts

```diff
@@ -43,10 +43,10 @@ export async function yesNoQuestion (options: QuickPickOptions, shouldThrow = fa
 	}
 }
 
-export async function inputBox (options: InputBoxOptions) {
+export async function inputBox (options: InputBoxOptions, optional?: boolean ) {
 	const input = await window.showInputBox(options);
 
-	if (!input) {
+	if (!input && !optional) {
```

Comment:
I do wonder if we need to improve the validation here, [the docs](https://code.visualstudio.com/api/references/vscode-api#window.showInputBox) state that the value returned is either `undefined` if cancelled (hitting `esc` or clicking outside the box), or the string value if enter is returned.

So maybe the checks needs to be `if (input !== undefined)`, I think then we could drop the `optional` parameter and the onus would be on the function caller to validate